web/src/features/dialog/components/select.tsx

```diff
@@ -23,16 +23,16 @@ const SelectField: React.FC<Props> = (props) => {
     <>
       <Box mb={3} key={`select-${props.index}`}>
         <Select
-          onChange={(e) => props.handleChange(e.target.value, props.index)}
-          defaultValue={props.row.default}
+          onChange={(e: any) => props.handleChange(e.target.value, props.index)}
```

Comment:
Why explicitly type this as `any`?

The type was already getting inferred, don't see a reason to explicitly type it.

Reply:
Should I use `React.ChangeEvent<HTMLSelectElement>` instead or just keep it implicitly as `any`?

Reply:
> Should I use `React.ChangeEvent<HTMLSelectElement>` instead or just keep it implicitly as `any`?

You should be able to just remove the type completely and it should get inherited.

Otherwise you can use `React.ChangeEvent<HTMLSelectElement>`.

Reply:
Without any explicit type, it is `any`, so I used the React one.